test/draw_line_string.test.js

```diff
@@ -137,7 +137,7 @@ test('draw_line_string render active line with 1 coordinate', t => {
     },
     geometry: {
       type: 'LineString',
-      coordinates: [[0, 0]]
+      coordinates: [[0, 0], [15, 15]]
```

Comment:
Oh, this test is just wrong? It should return nothing right, as its an invalid LineString?